src/docs/codelabs/extended-flex-widget.md

```diff
@@ -0,0 +1,597 @@
+If the combined width of `Row` and `Column`'s child widgets is 
+less than the total space on `Row` and `Column`'s main axes, 
+the children are laid out with extra space.
+
+`MainAxisSize.min`
+: Decreases the amount of space `Row` and `Column` can occupy on their main axes.
+`Row` and `Column`'s children are laid out without extra space. 
+
+{{site.alert.tip}}
+  `MainAxisSize.max` is the default value for the `mainAxisSize` property. 
+  If you don't specify another value, 
+  the default value is used, 
+  as shown in the previous example. 
+{{site.alert.end}}
+#### Example: Modify axis size
+{:.no_toc}
+{{site.alert.secondary}}
+{:.no_toc}
+
+  The following example explicitly sets `mainAxisSize` to its default value, `MainAxisSize.max`.
+
+  **1.** Click the **Run** button. 
+
+  **2.** Change the `mainAxisSize` property to `MainAxisSize.min`, and run again. 
+{{site.alert.end}}
+{% comment %}
+  Gist: https://gist.github.com/d852e4f07d6c87600fe8e0f186c7a31b
+{% endcomment %}
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=d852e4f07d6c87600fe8e0f186c7a31b&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
+
+### mainAxisAlignment property
+
+When `mainAxisSize` is set to `MainAxisSize.max`, 
+`Row` and `Column` might lay out their children with extra space. 
+The `mainAxisAlignment` property determines how `Row` and `Column` 
+can position their children in that extra space. 
+ `mainAxisAlignment` has six possible values: 
+
+`MainAxisAlignment.start`
+: Positions children near the beginning of the main axis. 
+(Left for `Row`, top for `Column`)
+
+`MainAxisAlignment.end`
+: Positions children near the end of the main axis. 
+(Right for `Row`, bottom for `Column`)
+
+`MainAxisAlignment.center`
+: Positions children at the middle of the main axis.
+
+`MainAxisAlignment.spaceBetween`
+: Divides the extra space evenly between children.
+
+`MainAxisAlignment.spaceEvenly`
+: Divides the extra space evenly between children and before and after the children.
+
+`MainAxisAlignment.spaceAround`
+: Similar to `MainAxisAlignment.spaceEvenly`,
+but reduces half of the space before the first child and after the last child 
+to half of the width between the children. 
+
+#### Example: Modify main axis alignment
+{:.no_toc}
+{{site.alert.secondary}}
+
+  The following example explicitly sets the `mainAxisAlignment` property to its default value, 
+  `MainAxisAlignment.start`.
+
+  **1.** Click the **Run** button.
+
+  **2.** Change `MainAxisAlignment.start` to `MainAxisAlignment.end`, and run again.
+{{site.alert.end}}
+{% comment %}
+  Gist: https://gist.github.com/cb8abed13f90a6a0c7a0ada6f15a09c9
+{% endcomment %}
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=cb8abed13f90a6a0c7a0ada6f15a09c9&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
+{{site.alert.tip}}
+  Before moving to the next section, change `MainAxisAlignment.end` to one of the other values.
```

Comment:
GLOBAL: "one of the other values" -> "another value"
(when you're talking about using another enum value)
(Other solutions are possible that use enum terminology... My beef with "one of the other values" is that it's a lot of words yet not precise enough to warrant them.)